File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int validheight = 0;
    while(validheight != 1){
        int h = get_int("Height: ");
        if(h >= 0 && h <= 23){
            for(int i = 0;i <= h-1; i++){
                for(int j = 0; j <h-i-1;j++)
                    printf(" ");
                for(int k = 0; k <= i; k++)
                    printf("#");
                printf("  ");
                for(int l = 0; l <= i; l++)
                    printf("#");
                printf("\n");
            }
            validheight = 1;
        }
    }

}

File: mario.py
from cs50 import get_int

validheight = 0;
while validheight != 1:
    print("Height:")
    h = get_int()
    if h < 0 or h > 23:
        break
    for i in range(h):
        print(" " * (h - i), end="")
        print("#" * (i + 1) + "  "+ "#" * (i + 1))
    validheight = 1
